### ISSUE TITLE
Design VAMC top task editorial experience: Policies page

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#3153]
@rachel-kauff @kaise-lafrai @kevwalsh @nataliedunbarCS  and possibly @jordanamello 

## Problem statement 
The governance and content design needs determined by #3025 require some UX design to provide the best possible editorial and governance experience for these important pages for Veterans. 

Some proposed principles:
* Editors should understand what they can edit and what they can't, and why. 
* Guidelines for X content types should  
* Content creation, content editing (existing content), content proofing

Candidate pages 
* See inventory at https://prod.cms.va.gov/admin/content/audit/vamc-top-task-pages  (also listed as subepics on the superepic #2721)

Candidate opportunities for design solutions
* A reusable set of design patterns for handling national vs local content at a page level. 
  * This could _potentially_ be applied for health service content too, although that one is slightly more complicated because it's national/regional/local. 
* Standardize the diversity of UX labels "owner" vs "section" vs "governance" (could become site-wide, but we may start with just these)

We may consider two user story maps, for different types of editors (VHA HQ vs VAMC)

## ACs

* We can begin technical discovery #3027

**Comment by kevwalsh** [comment]
@davidconlon 

https://balsamiq.cloud/s21kwcm/py46j7t/r1717

There are four wireframes for review

1. National edit
2. National proofing
3. Local edit
4. Local proofing

I'd pay the closest attention to the Local edit screen. There's a green box to the right of it that represents some slightly simpler implementation options, using rich text fields instead of lightly structured content (policy name, policy description)

I believe most of the details can be worked out as we approach the ACs for actual implementation, next week. For this week, our engineering team is focused more on the governance architecture than field level content modeling.

**Comment by davidconlon** [comment]
@kevwalsh 

**TL/DR:** I'm concerned about the bifurcation of the use of rich text fields and how we can ensure both national and local editors use the right semantic markup. This is already a big issue for 508.

**National Edit**
I think that this version may be confusing for editors in ensuring proper structure/markup of the content. 
e.g. "We've disabled headers in the intro paragraph, but use an H2 in top of page related policies, but not in Visitation policies because we've baked it into the template use an H3....
Also, Consider: In the edit note for "general visitation policy" a note to prompt VHA Central Office to notify the field that changes have been made so that those sites can verify that they are not duplicating or contradicting any information in the update.

**National proofing**
This makes sense, provided we help the editor understand proper semantic markup.

**Local Editors** 
 "Visitation Policy" (singluar, not plural) and do the same for "Other Policy"
Same comment as above: How do we ensure editors do proper semantic markup with similar yet different editorial experiences.
If we use the same modal for "visitation guideline" as "other policy" it could be confusing that in one case we want the description rich text to allow/encourage h3's and the other no headings or h4's?
I'd suggest that we lean into the MVP and the "other policy" require a plain text heading and a rich text field so that we keep it parallel for the editorial experience.
